packages/expo-application/ios/EXApplication/EXApplication.m

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+#import <UMCore/UMUtilities.h>
+#import <EXApplication/EXApplication.h>
+#import <UIKit/UIKit.h>
+
+@interface EXApplication ()
+
+
+@end
+
+@implementation EXApplication
+
+UM_EXPORT_MODULE(ExpoApplication);
+
+- (dispatch_queue_t)methodQueue
+{
+  return dispatch_get_main_queue();
+}
+
+UM_EXPORT_METHOD_AS(getIosIdForVendorAsync, getIosIdForVendorAsyncWithResolver:(UMPromiseResolveBlock)resolve rejecter:(UMPromiseRejectBlock)reject)
+{
+  if(NSClassFromString(@"UIDevice") && [UIDevice instancesRespondToSelector:@selector(identifierForVendor)]) {
```

Comment:
We probably don't need this check -- on all the devices we support `- [UIDevice identifierForVendor]` should work I think (or is that not the case)?

Reply:
It should work across all devices. I'll remove this check!